src/main.rs

```diff
@@ -35,6 +36,13 @@ fn main() {
         .unwrap();
     f.write_all(output.to_string().as_bytes()).unwrap();
 
+    if !&conf.ignore_lockfile {
+        Command::new("cargo")
+        .args(&["generate-lockfile", "--offline", "--manifest-path", &conf.manifest.to_string_lossy()])
```

Comment:
i found this command change more than just the current crate version

let's instead change it to `cargo build` it seems safer
```suggestion
        .args(&["build"])
```